tests/endtoend/eventhub_batch_functions/get_metadata_batch_triggered/function.json

```diff
@@ -0,0 +1,22 @@
+{
+  "scriptFile": "__init__.py",
+  "bindings": [
+    {
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req"
+    },
+    {
+      "type": "blob",
```

Comment:
Quick question - shouldn't the type be `eventHub`? Could you please describe the test a little in the pythondoc comments in the dunder init of that function?

Reply:
Even a one-liner would help get the context of the E2E tests.

Reply:
Good point, but unfortunately I cannot make comment in json file.
These tests are all following the flow of
1. metadata_output => an HTTP trigger generating EventHub Event from azure-eventhub SDK.
2. metadata_multiple => an EventHub trigger which will be executed when the newly generated Event arrives EventHub. Writing the event metadata to a blob storage.
3.  get_metadata_batch_triggered => an HTTP trigger for getting the EventHub Event metadata from the blob, responding back to the E2E testcase.

Reply:
I will add these comments in the .py file instead.